tutorials/clinical/10_ieeg_localize.py

```diff
@@ -0,0 +1,487 @@
+# Aligning the CT to the MR
+# =========================
+#
+# Let's load our CT image and visualize it with the T1 image.
+# As you can see, the CT is already aligned to the T1 image in this example
+# dataset.
+
+CT = nib.freesurfer.load(op.join(misc_path, 'seeg', 'sample_seeg_CT.mgz'))
+
+# make low intensity parts of the CT transparent for easier visualization
+CT_data = CT.get_fdata().copy()
+CT_data[CT_data < np.quantile(CT_data, 0.95)] = np.nan
+
+fig, axes = plt.subplots(1, 3, figsize=(12, 6))
+for ax in axes:
+    ax.axis('off')
+axes[0].imshow(T1.get_fdata()[T1.shape[0] // 2], cmap='gray')
+axes[0].set_title('MR')
+axes[1].imshow(CT.get_fdata()[CT.shape[0] // 2], cmap='gray')
+axes[1].set_title('CT')
+axes[2].imshow(T1.get_fdata()[CT.shape[0] // 2], cmap='gray')
+axes[2].imshow(CT_data[CT.shape[0] // 2], cmap='gist_heat', alpha=0.5)
+for ax in (axes[0], axes[2]):
+    ax.annotate('Subcutaneous fat', (110, 52), xytext=(100, 30),
+                color='white', horizontalalignment='center',
+                arrowprops=dict(facecolor='white'))
+for ax in axes:
+    ax.annotate('Skull (dark in MR, bright in CT)', (40, 175),
+                xytext=(120, 246), horizontalalignment='center',
+                color='white', arrowprops=dict(facecolor='white'))
+axes[2].set_title('CT aligned to MR')
+fig.tight_layout()
+
+
+###############################################################################
+# Let's unalign our CT data so that we can see how to properly align it.
+#
+# .. note::
+#     The hyperintense skull is actually aligned to the hypointensity between
+#     the brain and the scalp. The brighter area surrounding the skull in the
+#     MR is actually subcutaneous fat.
+
+def plot_overlay(image, compare, title, thresh=None):
+    """Define a helper function for comparing plots."""
+    image = nib.orientations.apply_orientation(
+        image.get_fdata().copy(), nib.orientations.axcodes2ornt(
+            nib.orientations.aff2axcodes(image.affine)))
+    compare = nib.orientations.apply_orientation(
+        compare.get_fdata().copy(), nib.orientations.axcodes2ornt(
+            nib.orientations.aff2axcodes(compare.affine)))
+    if thresh is not None:
+        compare[compare < np.quantile(compare, thresh)] = np.nan
+    fig, axes = plt.subplots(1, 3, figsize=(12, 4))
+    fig.suptitle(title)
+    for i, ax in enumerate(axes):
+        ax.imshow(np.take(image, [image.shape[i] // 2], axis=i).squeeze().T,
+                  cmap='gray')
+        ax.imshow(np.take(compare, [compare.shape[i] // 2],
+                          axis=i).squeeze().T, cmap='gist_heat', alpha=0.5)
+        ax.invert_yaxis()
+        ax.axis('off')
+
+
+# Make an affine to transform the image
+unalign_affine = np.array([
+    [-1.01, 0.02, -0.01, 128],
+    [0.01, -0.02, 1.02, -135],
+    [0.06, -0.99, -0.02, 140],
+    [0, 0, 0, 1]])
+CT_unaligned = resample_from_to(CT, (CT.shape, unalign_affine))
```

Comment:
Do you not have the original-unaligned CT? Or did it come aligned already? I'm inclined to have some form of "unaligned CT" (can be this one) as the version stored in the dataset if it reflects the more general case that most people will have. It's better than faking it in the tutorial itself.